apps/finance/contracts/Finance.sol

```diff
@@ -105,23 +102,20 @@ contract Finance is AragonApp, ERC677Receiver {
      * @notice Allows to send ETH from this contract to Vault, to avoid locking them in contract forever.
      */
     function escapeHatch() public payable {
-        // convert ETH to EtherToken
-        etherToken.wrapAndCall.value(this.balance)(address(this), "Adding Funds");
+        vault.deposit.value(msg.value)(address(0), msg.sender, 0, new bytes(0));
```

Comment:
Better to use an `ETH` constant than passing around `address(0)` everywhere

Reply:
Sure!